apps/react/src/features/genres/components/GenreCard/GenreCard.tsx

```diff
@@ -3,18 +3,26 @@ import { Genre } from '@js-camp/core/models/genre';
 
 import styles from './GenreCard.module.css';
 
-interface Props {
+type Props = {
 
 	/** Genre. */
 	readonly genre: Genre;
-}
+};
 
 /** Card with genre data. */
-const GenreCardComponent: FC<Props> = ({ genre }) => (
+const GenreCardComponent: FC<Props> = ({ genre }: Props) => (
```

Comment:
`FC<Props>` is enough.

```suggestion
const GenreCardComponent: FC<Props> = ({ genre }) => (
```

Reply:
Apparently, if I only use `FC<Props>`, and run the script `npm run react:lint`, I will have this error message: 

> Cannot read properties of undefined (reading 'typeAnnotation')
Occurred while linting .../camp-js-2024-man-pham/apps/react/src/features/genres/components/GenreCard/GenreCard.tsx:12
Rule: "jsdoc/require-param"
Pass --verbose to see the stacktrace.

So my VN team decided to put `{ genre }: Props` to prevent this error